src/sentry/api/endpoints/event_details.py

```diff
@@ -21,6 +22,26 @@ def get(self, request, event_id):
         is the event as it appears in the Sentry database and not the event
         ID that is reported by the client upon submission.
         """
+        use_snuba = options.get('snuba.events-queries.enabled')
+
+        if not use_snuba:
+            return self.get_legacy(request, event_id)
+
+        event = SnubaEvent.objects.from_event_id(event_id, project_id=None)
```

Comment:
We need the project id for snubaevent otherwise this query will take like a minute, and we won't have the data we need to look up the nodestore data. I think maybe that means this endpoint needs to change or be deprecated somehow.

Reply:
Yeah I'm not actually not sure where we even use it. Will try to add some logging to understand where this is happening.

I'm guessing we'll want to migrate these 2 as well https://github.com/getsentry/sentry/blob/master/src/sentry/api/urls.py#L1234=L1242